app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,9 @@
+class LikesController < ApplicationController
+  def new
+    post = Post.find(params[:post_id])
+    author = User.find(params[:user_id])
+    like = Like.new(author:, post:)
+
+    return unless like.save
+  end
```

Comment:
### [OPTIONAL]
Please follow the instructions mentioned in the Posts Controller. 👍